packages/paste-core/primitives/menu/package.json

```diff
@@ -1,7 +1,37 @@
 {
-  "name": "@twilio-paste/primitive-menu",
+  "name": "@twilio-paste/menu-primitive",
   "version": "0.0.2",
   "category": "interaction",
-  "status": "backlog",
-  "private": true
+  "status": "beta",
+  "description": "An unstyled and accessible basis upon which to build Menus.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && yarn compile",
+    "build:dev": "yarn clean && yarn compile:dev",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist && rm -rf tsconfig.build.tsbuildinfo && rm -rf .rpt2_cache",
+    "compile": "rollup -c --environment NODE_ENV:production",
+    "compile:dev": "rollup -cw --environment NODE_ENV:development",
+    "prepublishOnly": "yarn build",
+    "type-check": "tsc --noEmit"
+  },
+  "dependencies": {
+    "reakit": "^1.0.2"
```

Comment:
Once https://github.com/twilio-labs/paste/pull/446 is in can change to the library version as a peer dep

Reply:
Can do this now